src/components/Search/MainPage.js

```diff
@@ -0,0 +1,91 @@
+import React, {Component} from 'react';
+
+import { Route, Link } from "react-router-dom";
+
+import Header from '../Header/Header';
+import SearchBox from '../Search/SearchBox';
+import SearchResults from '../Search/SearchResults';
+
+export default class MainPage extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            showSearchButtonOnHeader: false,
+            movies: [],
+            titleActive: true,
+            sortByReleaseDate: true,
+            searchText: ''
+        };
+        this.titleClick = this.titleClick.bind(this);
+        this.genreClick = this.genreClick.bind(this);
+        this.searchClick = this.searchClick.bind(this);
+        this.releaseDateClick = this.releaseDateClick.bind(this);
+        this.ratingClick = this.ratingClick.bind(this);
+        this.valueChangeHandler = this.valueChangeHandler.bind(this);
+    }
+    componentDidMount() {
+    }
+    loadData() {

```

Comment:
This method could have a more descriptive name, you're searching the date not only loading.

Reply:
Done